test/vm.cpp

```diff
@@ -98,10 +98,10 @@ void FakeExtVM::push(mArray& a, u256 _v)
 mObject FakeExtVM::exportEnv()
 {
 	mObject ret;
-	ret["previousHash"] = toString(previousBlock.hash);
+	ret["previousHash"] = "0x"+toString(previousBlock.hash);
```

Comment:
addresses and hashes do not need to get prefixed by 0x. Only numbers which are given in hex format. Therefore all changes in this file can be reverted.